### DISCUSSION TITLE
如何使用正则在网页中查找

### DISCUSSION THREAD

**Discussion by batu1579** [gdh1995/vimium-c#870] [Q&A] [answered]
怎么才能用正则在网页中查找来着，我记得以前一开始搜索就能看到提示，现在怎么也找不到辣QAQ

**Comment by gdh1995** [comment, marked as answer]
\r 开启，或者在设置里勾选 “页内查找时默认将搜索词视为正则表达式” 来常开



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年2月21日(周二) 凌晨3:51
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: [gdh1995/vimium-c] 如何使用正则在网页中查找 (Discussion #870)




 
怎么才能用正则在网页中查找来着，我记得以前一开始搜索就能看到提示，现在怎么也找不到辣QAQ

**Reply by batu1579** [reply]
太棒啦，谢谢大大！！！ ヾ(≧▽≦*)o
